cmd/bb_browser/BUILD.bazel

```diff
@@ -1,13 +1,23 @@
 load("@com_github_buildbarn_bb_storage//tools:container.bzl", "container_push_official")
 load("@io_bazel_rules_docker//container:container.bzl", "container_image", "container_layer")
-load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_library")
+load("@io_bazel_rules_go//go:def.bzl", "go_binary", "go_embed_data", "go_library")
+
+go_embed_data(
```

Comment:
Oh, nice. I wasn't aware of the existence of `go_embed_data()`. Would you be interested in removing `assets/bundle_file.*` and patching up that part of bb-browser to also use that?

Reply:
Good point, done!